content/cloud/custom-domains-tutorial.md

```diff
@@ -0,0 +1,73 @@
+
+## Prerequisites
+
+To ensure the tutorial goes smoothly, please ensure you have the following: 
+
+* Most importantly, you must have a domain you own. If you don't have a domain, please visit a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google) to purchase one. 
+* A Fermyon Cloud account that is set up via your preferred [GitHub user account](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email)
+* A Spin application you would like to apply the custom domain to. If you aren't familiar with how to deploy an application on Fermyon Cloud, check out our [develop a Spin application](./develop.md) article to learn how to build a Spin application and then our [deploy a Spin application article](./deploy.md) for deployment instructions. 
+
+## Select Your Spin Application
+
+In the [Fermyon Cloud Dashboard](https://cloud.fermyon.com), navigate to your Spin application of choice and click the "Manage Domains" button.
+
+![DRAFT SCREENSHOT - app panel view](/static/image/custom-domains/app-panel-view.png)
+
+## Input Your Custom Domain
+
+Select the "Add custom domain" button and input your desired domain name. 
+
+![DRAFT SCREENSHOT - step 1](/static/image/custom-domains/Step-1.png)
+
+Please add the complete domain name (i.e., `www.example.com`) as Fermyon Cloud does not support redirects at this time. Click save to confirm you would like to add this domain name to Fermyon Cloud.
+
+![DRAFT SCREENSHOT - step 2](/static/image/custom-domains/Step-2.png)
+
+Up next, you'll need to configure your DNS records with your registrar to point at Fermyon's name servers (NS). 
+
+>>If you click "I'll do this later" you will be pausing the workflow and your *custom domain will not work yet* with your Spin application. You can navigate back to the "Manage Domains" workflow to resume your journey. 
+
+![DRAFT SCREENSHOT - step 3](/static/image/custom-domains/Step-3.png)
+
+## Update Your Domain Registrar With Fermyon Cloud Name Servers
+
+At this step, Fermyon Cloud will check to see if you've previously associated your domain with Fermyon Cloud before. If this is the first time Fermyon Cloud has seen the domain, you will need to visit your domain registrar to delegate your domain to Fermyon Cloud. Domain delegation is the process of reassigning nameservers for a domain name to another DNS provider, which allows said nameservers to handle all DNS requests and queries for that domain. 
```

Comment:
So if FC _has_ seen the domain before then I don't need to do anything?  The "in this step" paragraph doesn't really elaborate on this.
```suggestion
At this point, Fermyon Cloud checks to see if you've previously associated your domain with Fermyon Cloud before.

If you _have_ previously associated your domain, there's nothing more to do in this step!  Skip ahead to the next step.

If, however, this is the first time Fermyon Cloud has seen the domain, you will need to visit your domain registrar to delegate your domain to Fermyon Cloud. Domain delegation is the process of reassigning nameservers for a domain name to another DNS provider, which allows said nameservers to handle all DNS requests and queries for that domain. 
```

(not sure I've brought out the contrast, and it would be better if the "not done before" flow was the main one... but hopefully gives you ideas)

Reply:
I love this re-jigging. Much clearer.